.github/workflows/pip_install.yml

```diff
@@ -9,12 +9,16 @@ jobs:
       fail-fast: false
       matrix:
         python: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11']
+        setuptools-version: ['39.2.0', '59.6.0', '68.2.2']
```

Comment:
Not all of these are probably compatible with all those Python versions...

Ideally we just add one extra combo to cover the fix in #86?

Reply:
indeed, i first wanted to check which combos are ok and which are not.

now that it's clear that it's setuptools `39.2.0` is the problem, we can reduce the number of tests